### PR TITLE
chore: Remove examples/shim

### PR DESCRIPTION
**Pull request by amychisholm03**
Removed the examples/shim folder, added instrumentation.md to redirect users, and updated jsdoc

### CODE REVIEW

**Review by bizob2828 — CHANGES_REQUESTED** [review]
You need to add a `jsdoc.json` so we can have a page in https://newrelic.github.io/node-newrelic/ that will publish this markdown. 

If you run `npm run public-docs` this now fails because it's expecting `examples/shim`. you can just update the `public-docs` script to include `documentation/examples.md`

File: documentation/examples.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# node-newrelic examples
+
+We have moved our examples and tutorials over to [newrelic-node-examples](https://github.com/newrelic/newrelic-node-examples) as more robust applications. Here are some quick links for implementing custom instrumentation and using our shim API:
```
Comment (bizob2828) [inline documentation/examples.md:3]: It looks like to get just this as a tutorial you'll need it in its own folder. I would move this to `examples/instrumentat.md` and then add `jsconf.json` like this

```
{
  "instrumentation": {
    "title": "Instrumentation Examples"
  }
}
```

Comment (amychisholm03) [reply documentation/examples.md:3]: Oops just saw this I'll redo what I did

**Review by jsumners-nr — APPROVED** [review]
Looks good over here. `npm run public-docs` results in the desired output as far as I can tell.

File: examples/instrumentation.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+* [backgroundTransactions](https://github.com/newrelic/newrelic-node-examples/tree/main/custom-instrumentation/background-transactions) - example application that uses the newrelic API to create [background transactions](https://newrelic.github.io/node-newrelic/API.html#startBackgroundTransaction)
+* [segments](https://github.com/newrelic/newrelic-node-examples/tree/main/custom-instrumentation/segments) - example application that demonstrates how to use the [newrelic.startSegment API](https://newrelic.github.io/node-newrelic/API.html#startSegment) in a variety of cases: callback-based, promise-based, asyncronously, and syncronously
+* [distributed tracing](https://github.com/amychisholm03/newrelic-node-examples/blob/main/custom-instrumentation/distributed-tracing) - example application that demonstrates distributed tracing
+
+To request additional examples, please [file an issue](https://github.com/newrelic/node-newrelic/issues)!
```
Comment (bizob2828) [inline examples/instrumentation.md:14]:
```suggestion
To request additional examples, please [file an issue](https://github.com/newrelic/newrelic-node-examples/issues)!
```

we probably want them to add the issue to the examples repo

File: examples/instrumentation.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# node-newrelic examples
```
Comment (bizob2828) [inline examples/instrumentation.md:1]: not sure you need this. because when it renders in a tutorial it has a title

**Review by bizob2828 — COMMENTED** [review]
found one thing

File: examples/instrumentation.md (hunk 3 above)
Comment (bizob2828) [reply examples/instrumentation.md:1]: this header is still there

Comment (bizob2828) [inline examples/instrumentation.md:1]:
```suggestion
```

**Review by bizob2828 — CHANGES_REQUESTED** [review]

**Review by bizob2828 — APPROVED** [review]

**Pull request merged**